deployments/docker-compose.yaml

```
@@ -1,0 +1,86 @@
+
+volumes:
+  pgdata:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
+      device: ./../pgdata
+
+networks:
+  perseus:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)